src/components/Calculator.js

```diff
@@ -1,62 +1,60 @@
-    this.obj = {};
-  }
+let obj = {};
+
+const Calculator = () => {
+  const [output, setOutput] = useState(0);
 
-  handleButtonClick = (buttonName) => {
-    this.obj = calculate(this.obj, buttonName);
+  const handleButtonClick = (buttonName) => {
+    obj = calculate(obj, buttonName);
 
-    if (this.obj.operation && !this.obj.next) {
-      this.setState({ output: this.obj.operation });
-    } else if (this.obj.next) {
-      this.setState({ output: this.obj.next });
-    } else if (this.obj.total) {
-      this.setState({ output: this.obj.total });
-    } else {
-      this.setState({ output: 0 });
+    let outStr = '';
+    if (obj.total) {
+      outStr += obj.total;
+    }
+    if (obj.operation) {
+      outStr += obj.operation;
     }
-  }
+    if (obj.next) {
+      outStr += obj.next;
+    }
+    if (outStr === '') {
+      outStr += '0';
+    }
+
+    setOutput(outStr);
+  };
 
-  render() {
-    const { output } = this.state;
-    return (
-      <div className="calculator">
-        <Result output={output} />
-        <div className="numbers-oberations">
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="AC" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="+/-" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="%" />
-          <Button handleButtonClick={this.handleButtonClick} className="last" buttonName="&divide;" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="7" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="8" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="9" />
-          <Button handleButtonClick={this.handleButtonClick} className="last" buttonName="x" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="4" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="5" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="6" />
-          <Button handleButtonClick={this.handleButtonClick} className="last" buttonName="-" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="1" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="2" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="3" />
-          <Button handleButtonClick={this.handleButtonClick} className="last" buttonName="+" />
-          <Button handleButtonClick={this.handleButtonClick} className="zero-btn" buttonName="0" />
-          <Button handleButtonClick={this.handleButtonClick} className="" buttonName="." />
-          <Button handleButtonClick={this.handleButtonClick} className="last" buttonName="=" />
-        </div>
+  return (
+    <div className="calculator">
+      <Result output={output} />
```

Comment:
- [OPTIONAL] The display of your calculator is left aligned instead of right aligned as in the design, please kindly fix that.   See image. 

![left-aligned](https://user-images.githubusercontent.com/65852621/143410874-08a0eafd-c383-4107-b63b-22f40a748185.png)